pages/mesosphere/dcos/1.13/cli/command-reference/dcos-job/dcos-job-schedule-show/index.md

```diff
@@ -16,7 +16,7 @@ The `dcos job schedule show` command allows you to view a job schedule.
 # Usage
 
 ```bash
-dcos job schedule show <job-id> [--json]
+dcos job show <job-id> [--json]
```

Comment:
@janisz This one should have been kept as `dcos schedule show`.

Reply:
Good catch!